### ISSUE TITLE
tasks loading error on gcloud run

### ISSUE THREAD

**Issue by fishyboy** [heartexlabs/label-studio#360]
**Describe the bug**
deployed on google cloud with the one click button on readme.md page, a cloud run instance is successfully created. but it seems that this instance is always denied to read a gs bucket file. every time i click on tasks it shows following error:

Tasks loading error
Error calling the IAM signBytes API: b'{\n "error": {\n "code": 403,\n "message": "The caller does not have permission",\n "status": "PERMISSION_DENIED"\n }\n}\n'

**To Reproduce**
1. Go to 'TASKS' and configure cloud bucket correctly
2. TASKS will show the correct number of images in cloud bucket
3. give allUsers cloud admin permission
4. refresh page tasks to see error

**Expected behavior**
i suppose i should have seen images but there wasn't any

**Screenshots**

**Environment (please complete the following information):**
 - OS: ubuntu
 - Browser chrome

**Additional context**

**Comment by makseq** [comment]
Can you tell more about this step: "give allUsers cloud admin permission"? 
What permissions were before this step? Can you send screenshot - where you change these permissions in gcloud?

**Comment by fishyboy** [comment]
> Can you tell more about this step: "give allUsers cloud admin permission"?
> What permissions were before this step? Can you send screenshot - where do you change these permissions in gcloud?

Thanks for your reply.

After the one click deployment on gcloud, I created a task based on my gs bucket right away without changing any permission policy of anything. The IAM 403 denied error appeared right after that. 

I supposed that "The caller does not have permission" was due to lack of permission for the requested gs bucket so I changed permission configuration in my gs bucket page and let allUsers to be Cloud Storage Admin (so my gs bucket went public, and normally any request from anyone should be accepted and not denied. ). That didn't work. 

I've also looked into the permission config page of cloud run but I don't know what to do

Basically, I don't know what permission is required by LabelStudio to load and show my images on gs bucket
